styles/styles.css

```diff
@@ -68,6 +68,7 @@
   --color-black: #000;
   --color-navy: #133b5c;
   --color-blue: #006cb4;
+  --color-honolulu-blue: #016cb4;
```

Comment:
I think this is the wrong color. In my browser it's showing me the same color already defined as `--color-blue`